@xen-orchestra/web-core/lib/components/ui/dropdown/UiDropdown.vue

```diff
@@ -0,0 +1,213 @@
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { toVariants } from '@core/utils/to-variants.util'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faAngleRight, faLock } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+export type DropdownAccent = 'normal' | 'brand' | 'success' | 'warning' | 'danger'
+
+const {
+  accent,
+  disabled: _disabled,
+  locked,
+  selected,
+  hover,
+} = defineProps<{
+  accent: DropdownAccent
+  disabled?: boolean
+  locked?: boolean
+  selected?: boolean
+  hover?: boolean
+  checkbox?: boolean
+  subMenuIcon?: boolean
+  icon?: IconDefinition
+  info?: string
+}>()
+
+const disabled = computed(() => _disabled || locked)
+
+const className = computed(() =>
+  toVariants({
+    accent,
+    disabled: disabled.value,
+    selected,
+    hover,
+  })
+)
+</script>
+
+<style lang="postcss" scoped>
+.ui-dropdown {
+  display: flex;
+  align-items: center;
+  padding: 1.2rem;
+  gap: 0.8rem;
+  height: 4.5rem;
+
+  &.accent--normal {
+    color: var(--color-neutral-txt-primary);
+    background-color: var(--color-neutral-background-primary);
+
+    &.selected {
+      background-color: var(--color-brand-background-selected);
+    }
+
+    &:is(:hover, .hover) {
+      background-color: var(--color-brand-background-hover);
+    }
+
+    &:active {
+      background-color: var(--color-brand-background-active);
+    }
+
+    &.disabled {
+      color: var(--color-neutral-txt-secondary);
+      background-color: var(--color-neutral-background-disabled);
+    }
+  }
+
+  &.accent--brand {
+    color: var(--color-brand-txt-base);
+    background-color: var(--color-neutral-background-primary);
+
+    &.selected {
+      color: var(--color-brand-txt-base);
+      background-color: var(--color-brand-background-selected);
+    }
+
+    &:is(:hover, .hover) {
+      color: var(--color-brand-txt-hover);
+      background-color: var(--color-brand-background-hover);
+    }
+
+    &:active {
+      color: var(--color-brand-txt-active);
+      background-color: var(--color-brand-background-active);
+    }
+
+    &.disabled {
+      color: var(--color-neutral-txt-secondary);
+      background-color: var(--color-neutral-background-disabled);
+    }
+  }
+
+  &.accent--success {
+    color: var(--color-success-txt-base);
+    background-color: var(--color-neutral-background-primary);
+
+    &.selected {
+      color: var(--color-success-txt-base);
+      background-color: var(--color-success-background-selected);
+    }
+
+    &:is(:hover, .hover) {
+      color: var(--color-success-txt-hover);
+      background-color: var(--color-success-background-hover);
+    }
+
+    &:active {
+      color: var(--color-success-txt-active);
+      background-color: var(--color-success-background-active);
+    }
+
+    &.disabled {
+      color: var(--color-neutral-txt-secondary);
+      background-color: var(--color-neutral-background-primary);
```

Comment:
It should be:
```suggestion
      background-color: var(--color-neutral-background-disabled);
```
The same applies for `warning` and `success` variants